src/SDKs/Cdn/Management.Cdn/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CDN Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CDN.")]
 
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("5.0.0.0")]
```

Comment:
@prakharsharma10 AssemblyVersion and AssemblyFileVersion should match your nuget version.
This helps in maintain the versioning story same.
Change AssemblyFileVersion to 4.0.1.0 and AssemblyVersion to 4.0.0.0

Reply:
Thanks @shahabhijeet! Updated the versions.